.gitignore

```diff
@@ -1,3 +1,5 @@
+# Intellij IDEA
+.idea
 
```

Comment:
This is noise, please remove this from the pull request.

Reply:
It is for do not commit ( ignore ) the files from Golang IDEA as it is done for the other things there. If the dev user IDEA files from IDEA that should not have been trying to be committed.

Reply:
I do understand what the change to the `.gitignore` file does. It is just unrelated to the commit message, pr header, and purpose of the pull request. IMO a pull request to this repo should only include changes directly related to that pull request. If you would like to suggest an update to the `.gitignore` please open a separate pull request for that purpose.

Reply:
Ok I catcher your point. Tks for the clarifications.

Reply:
Done ..